### PR TITLE
Implement @reatom/observable

### LINKED ISSUE #119: Add "@reatom/observable" package
We need "@reatom/observable" package for easy integrations with other observable libraries.

### PR DESCRIPTION
**Pull request by Wroud**
Close #119

### CODE REVIEW

File: packages/observable/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# @reatom/observable
+
```
Comment (belozer) [inline packages/observable/README.md]: Needs simple description about package

File: packages/observable/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@reatom/observable",
+  "private": false,
+  "version": "1.0.0",
```
Comment (belozer) [inline packages/observable/package.json]: Please set 0.0.0 @artalar will fixed it

File: packages/observable/package.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  "jest": {
+    "testMatch": [
+      "**/test/**/*.[jt]s?(x)"
+    ]
+  },
+  "devDependencies": {
+    "cross-env": "^5.2.1",
+    "ts-node": "^8.3.0"
+  },
+  "dependencies": {
```
Comment (belozer) [inline packages/observable/package.json]: peerDependencies

File: packages/observable/package.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    "/package.json",
+    "/src/index.ts",
+    "/build"
+  ],
+  "jest": {
+    "testMatch": [
+      "**/test/**/*.[jt]s?(x)"
+    ]
+  },
+  "devDependencies": {
```
Comment (belozer) [inline packages/observable/package.json]: Needs add `"@reatom/core": "^1.0.0-rc15"`

File: packages/observable/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
+import { Store, Atom, Action } from '@reatom/core'
+
+export type ActionOrValue<T> = T extends undefined ? Action<any, string> : T
+
+export interface Observable<T> {
+  /**
+   * Subscribes observer for Store or Atom<T> updates
+   *
+   * @return {Subscription}
```
Comment (belozer) [inline packages/observable/src/index.ts]: Maybe is better use `@returns`?

Comment (Wroud) [reply packages/observable/src/index.ts]: Why? I think it's not matter

Comment (belozer) [reply packages/observable/src/index.ts]: In jsdocs `@return` and `@returns` is equivalent code

But it's about meaning when reading code.

Example with translete to russian:
```
return - вернуть
returns - возвращается
```

Comment (belozer) [reply packages/observable/src/index.ts]: This is not critical part.You don't have to edit it.

**Pull request merged**